group-coordinator/group-coordinator-api/src/main/java/org/apache/kafka/coordinator/group/api/assignor/SubscribedTopicDescriber.java

```diff
@@ -38,14 +36,4 @@ public interface SubscribedTopicDescriber {
      *         or -1 if the topic Id does not exist.
      */
     int numPartitions(Uuid topicId);
-
-    /**
-     * Returns all the available racks associated with the replicas of the given partition.
-     *
-     * @param topicId       Uuid corresponding to the partition's topic.
-     * @param partition     Partition Id within topic.
-     * @return The set of racks corresponding to the replicas of the topic's partition.
-     *         If the topic Id does not exist, an empty set is returned.
-     */
-    Set<String> racksForPartition(Uuid topicId, int partition);
```

Comment:
We cannot remove this from the public interface. Let’s return an empty set for now.